### DISCUSSION TITLE
Playwright Test Runner：folio

### DISCUSSION THREAD

**Discussion by roughsoft** [getgauge/taiko#1707] [Ideas]
**Is your feature request related to a problem? Please describe.**
folio+taiko is the same as jest+tailo
**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by zabil** [comment]
Taiko should work with folio as it works with jest. 
Are you facing any issue during the integration?

**Comment by roughsoft** [comment]
need some work 
folio is more power and more customable than jest.

**Comment by roughsoft** [comment]
I suggest making some example project.like these:
https://github.com/microsoft/playwright-test
We need taiko-folio-test

**Comment by zabil** [comment]
As you are experimenting with it. Would you be willing to create a sample project and raise a PR?

**Comment by roughsoft** [comment]
I can try it.
